PubnativeLite/PubnativeLite/Utils/Header Bidding/HyBidHeaderBiddingUtils.m

```diff
@@ -67,4 +67,27 @@ + (NSString *)eCPMFromAd:(HyBidAd *)ad withDecimalPlaces:(HyBidKeywordMode)decim
     }
 }
 
++ (NSString *)createAppMonetHeaderBiddingKeywordsStringWithAd:(HyBidAd *)ad
```

Comment:
There's a method missing for interstitial, no?

Reply:
https://github.com/pubnative/pubnative-hybid-android-sdk/blob/feature/OMA-832-hybid-extension/hybid.sdk/src/main/java/net/pubnative/lite/sdk/utils/HeaderBiddingUtils.java#L149

Reply:
@eros902002 correct. Yesterday and today I was focused only on the banner side. That's why I skipped here any interstitional method :)
But let me add it 👍🏻